charts/home-assistant/templates/deployment.yaml

```diff
@@ -336,7 +331,7 @@ spec:
               protocol: TCP
           env:
             - name: HA_URL
-              value: "http://localhost:{{ .Values.service.port }}"
+              value: "http://localhost:8123"
             {{- if .Values.appdaemon.ingress.enabled }}
             - name: DASH_URL
```

Comment:
This one might need to stay as the bind address otherwise it won't listen correctly. I'll have to test https://github.com/AppDaemon/appdaemon/blob/3e141b15b28de9c60fe6e9481581d2748e620cae/dockerStart.sh#L37